avocado/plugins/spawners/podman.py

```diff
@@ -97,21 +174,14 @@ async def spawn_task(self, runtime_task):
             runtime_task.task.status_services[0].uri = mounted_status_server_socket
 
         task = runtime_task.task
-        entry_point_cmd = '/tmp/avocado-runner'
         entry_point_args = task.get_command_args()
         entry_point_args.insert(0, "task-run")
-        entry_point_args.insert(0, entry_point_cmd)
+        entry_point_args.insert(0, "avocado.core.nrunner")
+        entry_point_args.insert(0, "-m")
+        entry_point_args.insert(0, python_binary)
```

Comment:
Is there a reason to insert in reverse order rather than simply appending the various arguments?

Reply:
No, there's not... I just followed the previous code way too deep.  I guess the previous code with a single insertion hid itself in plain sight, but I don't like the "RPN" approach here :)